drivers/usb/usbuhci_new/roothub.c

```diff
@@ -0,0 +1,470 @@
+#include "usbuhci.h"
+
+//#define NDEBUG
+#include <debug.h>
+
+VOID
+NTAPI
+UhciRHGetRootHubData(IN PVOID uhciExtension,
+                     IN PVOID rootHubData)
```

Comment:
This function should use PUSBPORT_ROOT_HUB_DATA directly. I'll push a change to fix this in usbport and usbohci shortly.

Reply:
I will fix this at the last minute or after fixes in usbport, so that the builder does not give errors.

Reply:
Sounds good. Let me know if you want me to get something into master before this is merged.